setup.cfg

```diff
@@ -17,7 +17,7 @@ project_urls =
 [options]
 install_requires =
     rsconnect-python>=1.7.1
-    notebook
+    notebook>=6.1.5,<7.0.0
```

Comment:
Do we have an open issue for compatibility with notebook v7?

Reply:
New issue filed: https://github.com/rstudio/rsconnect-jupyter/issues/323